documentation/modules/operators/ref-operator-cluster-feature-gate-releases.adoc

```diff
@@ -22,7 +22,7 @@ Alpha and beta stage features are removed if they do not prove to be useful.
 * The `UseStrimziPodSets` feature gate moved to GA stage in Strimzi 0.35 and the support for StatefulSets is completely removed. It is now permanently enabled and cannot be disabled.
 * The `StableConnectIdentities` feature gate moved to GA stage in Strimzi 0.39.
   It is now permanently enabled and cannot be disabled.
-* The `UseKRaft` feature gate is available for development only and does not currently have a planned release for moving to the beta phase.
+* The `UseKRaft` feature gate is in beta phase and is enabled by default.
```

Comment:
You use `stage` for the others and just here you use `phase`. Fine with me if it is intentional -> just wanted to point it out.